MantidPlot/src/Graph.cpp

```diff
@@ -968,6 +968,10 @@ void Graph::enableAutoscaling(bool yes) {
       d_plot->setAxisScaleDiv(i, *d_plot->axisScaleDiv(i));
     }
   }
+  // Propagate this to spectrogram
+  if (spectrogram()) {
```

Comment:
There's an isSpectrogram() method which would have been a more appropriate check that checking for a null pointer.